docs/latest/essentials/getting-started.md

```diff
@@ -10,11 +10,13 @@ git clone git@github.com:OHIF/Viewers.git
 
 or [Download the latest Master as a ZIP File](https://github.com/OHIF/Viewers/archive/master.zip).
 
-## Test the archive against our public DICOM server
+## Run the OHIF Viewer application against our public DICOM server
+You can either spin up locally or run with Docker. 
+
+### running locally
 
-## Set up and test the OHIF Viewer application:
 1. [Install Node.js](https://nodejs.org/en/)
-2. Open a new terminal tab in the `ohif-viewer` directory and install all dependency packages via NPM
+2. Open a new terminal tab, go under `ohif-viewer` directory and install all dependency packages via `yarn`
 
   ````bash
   cd Packages-react/ohif-viewer
```

Comment:
This location is no longer accurate. The `Packages-react/ohif-viewer/` folder has been moved to `root`

Reply:
We also officially recommend `yarn` over `npm`, as it's what our dependent packages and CI are using to build.